qiskit/visualization/pulse/matplotlib.py

```diff
@@ -598,7 +599,7 @@ def _draw_table(self, figure, channels: Dict[Channel, EventsOutputChannels], dt:
                 # pylint: enable=unbalanced-tuple-unpacking
                 time, ch_name, data_str = data
                 # item
-                cell_value[r][3 * c + 0] = "t = %s" % time * dt
+                cell_value[r][3 * c + 0] = "t = %s" % time * dt  # TODO: worng operator priority?
```

Comment:
oh! good catch. let's fix it directly
```suggestion
                cell_value[r][3 * c + 0] = "t = %s" % (time * dt)
```